Makefile

```diff
@@ -23,7 +25,10 @@ install-tools:
 	hack/install-kubebuilder-tools.sh
 
 test: build install-tools
-	hack/test-go.sh
+	hack/test-go.sh --skip-static-check
```

Comment:
wait, isn't this going to skip the static check but, we actually want to run it here, right?

Reply:
no since the default is false

Reply:
I tested this on my local the logic behaves as expected, if you dont pass in skip-static-check arg it will error out, the way its written currently its not an optional var